napalm_ios/ios.py

```diff
@@ -73,6 +73,8 @@ def __init__(self, hostname, username, password, timeout=60, optional_args=None)
         self.password = password
         self.timeout = timeout
 
+        self.transport = optional_args.get('transport', 'ssh')
```

Comment:
@dbarrosop FYI, here you recommended calling it `transport` and the PR implemented it that way.

https://github.com/napalm-automation/napalm-ios/issues/132

IMO we should call it `transport` as it for any netmiko-based driver we will have the same pattern.